framework/source/class/qx/ui/tabview/Page.js

```diff
@@ -54,6 +54,22 @@ qx.Class.define("qx.ui.tabview.Page",
       this.setIcon(icon);
     }
 
+    // ARIA attrs
+    var btn = this.getButton();
+    var pageId = "page-" + this.toHashCode();
+    var btnId = "btn-" + pageId + btn.toHashCode();
+    var contentEl = this.getContentElement();
+    contentEl.setAttribute("id", pageId); 
```

Comment:
Same here - @ArturWesner, what are the ids needed for? would the look-up using the "data-qx-object-id" attribute be ok, too? This might be a bit slower than the DOM ids, but would not clutter the id space of qooxdoo users, and would be more "qooxdoo-ish".

Reply:
Have a look at https://qooxdoo.org/documentation/6.0/#/core/object_id?id=universal-object-id - it's a really cool and versatile system.

Reply:
Using qx ObjectIds would be indeed neat, but that's not possible sadly. The ARIA attributes depend on DOM ids and this can't be changed. In the issue's marked code the page and btn need DOM ids, so i can set `aria-labelledBy` and `aria-controls`. These tell the screenreader and user that the page is controlled and labelled by the btn text.
In case of the table i need the cells to have ids so that i can set the attribute `aria-activedescendant`. This attribute is used to tell the screenreader which element has focus without moving the browser focus. This focus handling can be found in many places: tables, selectboxes, comboboxes, ... .

Reply:
Ok sure, that rules out using qx-object-id attributes. So no objections from my side